src/dsync/dsync.c

```diff
@@ -1197,14 +1206,20 @@ static int dsync_strmap_compare_lite(
         uint64_t src_mtime_nsec = mfu_flist_file_get_mtime_nsec(src_compare_list, idx);
         uint64_t dst_mtime      = mfu_flist_file_get_mtime(dst_compare_list, idx);
         uint64_t dst_mtime_nsec = mfu_flist_file_get_mtime_nsec(dst_compare_list, idx);
+        char srcmtimeString[100];
+        char dstmtimeString[100];
+        strftime(srcmtimeString, sizeof(srcmtimeString), "%c", localtime(&src_mtime));
+        strftime(dstmtimeString, sizeof(dstmtimeString), "%c", localtime(&dst_mtime));
 
         /* if size or mtime is different, we assume the file contents are different */
         if ((src_size != dst_size) ||
-            (src_mtime != dst_mtime) || (src_mtime_nsec != dst_mtime_nsec))
+            (src_mtime != dst_mtime) || (comp_mtime_nsec ? (src_mtime_nsec != dst_mtime_nsec):false))
         {
             /* update to say contents of the files were found to be different */
             dsync_strmap_item_update(src_map, name, DCMPF_CONTENT, DCMPS_DIFFER);
             dsync_strmap_item_update(dst_map, name, DCMPF_CONTENT, DCMPS_DIFFER);
+            
+            MFU_LOG(MFU_LOG_WARN, "file: %s src_mtime: %s dst_mtime: %s\n", name, srcmtimeString, dstmtimeString);
```

Comment:
"Warning" or MFU_LOG_WARN is for cases where something might be broken broken, for example data may not be copied correctly or files may be lost.    In this case, are you just telling the user why a file was copied?

If so, look for other uses of MFU_LOG();  MFU_LOG_DBG or MFU_LOG_INFO are probably more appropriate.

Reply:
Debug statement left over. Taking it out.